packages/devtools/web/index.html

```diff
@@ -8,6 +8,154 @@
 
 <html>
 <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-26406144-33"></script>
+
+    <dialog open id="devtools_analytics" class="ga-dialog" style="width: 100%;">
+      <p>Welcome to Flutter DevTools</p>
+      <span class="ga-collection" disabled>
+        Flutter DevTools anonymously reports features usage statistics and basic crash reports to Google in order
+        to help Google contribute improvements to Flutter DevTools over time.
+        <br/>
+        See Google's private policy
+      </span>
+      <a id="devtools-ga-link" href="https://www.google.com/intl/en/policies/privacy" target="_blank" tabindex="1">https://www.google.com/intl/en/policies/privacy</a>
+      <br/><br/>
+      <p>Do you accept statistics collection for DevTools?</p>
+      <button id="devtools-ga-accept" value="accept" tabindex="2">I Accept</button>
+      <button id="devtools-ga-do-not-accept" value="do-not-accept" tabindex="3" autofocus>I Do Not Accept</button>
+    </dialog>
+
+    <script>
+      // URI parameter to control DevTools analytics collection:
```

Comment:
split this script out into a separate js file?

Reply:
Done.